controllers/example_controller/keyboard_controller.py

```diff
@@ -43,26 +61,34 @@ def print_distance_sensors(robot: Robot) -> None:
     else:
         while key != NO_KEY_PRESSED:
 
-            if key == ord('W'):
+            if key == key_forward:
                 R.motors[0].m0.power = 50
                 R.motors[0].m1.power = 50
 
-            elif key == ord('S'):
+            elif key == key_boost + key_forward:
+                R.motors[0].m0.power = 100
+                R.motors[0].m1.power = 100
+
+            elif key == key_reverse:
+                R.motors[0].m0.power = -50
+                R.motors[0].m1.power = -50
+
+            elif key == key_boost + key_reverse:
                 R.motors[0].m0.power = -50
                 R.motors[0].m1.power = -50
```

Comment:
Should these not be -100 as we're in boost mode?
```suggestion
                R.motors[0].m0.power = -100
                R.motors[0].m1.power = -100
```